pallets/registrar/src/lib.rs

```diff
@@ -62,19 +62,23 @@ pub mod pallet {
     pub struct Pallet<T>(_);
 
     #[pallet::genesis_config]
-    #[derive(Default)]
-    pub struct GenesisConfig {
+    #[derive(DefaultNoBound)]
+    pub struct GenesisConfig<T: Config> {
         /// Para ids
-        pub para_ids: Vec<(ParaId, ContainerChainGenesisData, Vec<Vec<u8>>)>,
+        pub para_ids: Vec<(
+            ParaId,
+            ContainerChainGenesisData<T::MaxLengthTokenSymbol>,
+            Vec<Vec<u8>>,
+        )>,
     }
 
     #[pallet::genesis_build]
-    impl<T: Config> GenesisBuild<T> for GenesisConfig {
+    impl<T: Config> GenesisBuild<T> for GenesisConfig<T> {
         fn build(&self) {
             // Sort para ids and detect duplicates, but do it using a vector of
             // references to avoid cloning the genesis data, which may be big.
             let mut para_ids: Vec<&_> = self.para_ids.iter().collect();
-            para_ids.sort();
+            para_ids.sort_by(|a, b| a.0.cmp(&b.0));
```

Comment:
was there any reason to change this?

Reply:
Yes, because I had to remove the `Ord` and `PartialOrd` traits because substrate doesn't provide `OrdNoBounds` and `PartialOrdNoBounds`. That may be useful so I will open a feature request upstream.